src/nvim/diff.c

```diff
@@ -1834,28 +1833,30 @@ int diffopt_changed(void)
   return OK;
 }
 
-/// Return TRUE if 'diffopt' contains "horizontal".
+/// Return true if 'diffopt' contains "horizontal".
 ///
-/// @return TRUE if 'diffopt' contains "horizontal"
-int diffopt_horizontal(void)
+/// @return true if 'diffopt' contains "horizontal"
```

Comment:
That was my thought as well, but it seemed that people wanted "full docs" so I went all out...

I would be happy to remove them as they seem redundant to me as well.